app/settings/page.tsx

```diff
@@ -24,16 +48,60 @@ function SettingsPage() {
     <>
       <Card>
         <CardHeader>
-          <CardTitle>Dev Area</CardTitle>
+          <CardTitle>Profile picture</CardTitle>
         </CardHeader>
         <CardContent className="space-y-4">
-          <div className="flex space-x-4">
-            <Button
-              variant="secondary"
-              onClick={() => createDefaultTaskSpecs()}
+          <div className="col-span-6 ml-2 sm:col-span-4 md:mr-3">
+            <input
```

Comment:
I'm using shadcn and tailwind for these ui elements, use `<Input/>` rather than `<input/>` and use tailwind instead of inline styles